fastlane/lanes/release.rb

```diff
@@ -463,9 +450,9 @@
     app = get_app_name_option!(options)
     app_values = APP_SPECIFIC_VALUES[app.to_sym]
     Dir.glob(File.join(app_values[:metadata_dir], 'android', '*', 'changelogs', '*')).each do |file|
-      if Integer(File.basename(file, '.*')) < Integer(options[:build])
-        File.delete(file)
-      end rescue puts "Cannot delete file #{file}"
+      File.delete(file) if Integer(File.basename(file, '.*')) < Integer(options[:build])
+    rescue StandardError
+      puts "Cannot delete file #{file}"
```

Comment:
Out of scope, but what do you think of relying on Fastlane's UI management?

```suggestion
      UI. error("Could not delete file #{file}.")
```

Reply:
Good idea! Done here: cc10f9df0b89255fcf721a723d5c90359f32a325